tests/wpt/mozilla/meta-layout-2020/mozilla/localeCompare.html.ini

```diff
@@ -1,4 +1,2 @@
 [localeCompare.html]
   bug: https://github.com/servo/servo/issues/25802
-  [localeCompare should return the same as other browsers, even though it's implementation-dependent]
-    expected: FAIL
```

Comment:
This change will need to be reverted. It has to do with locale settings on the CI machine.